CMakeLists.txt

```diff
@@ -614,3 +614,6 @@ add_tags(ctags
   EXCLUDE_OPTS ${CTAG_EXCLUDES}
   EXCLUDES "*.js" "*.css")
 add_custom_target(tags DEPENDS ctags)
+
+option(WITH_LIBRADOSSTRIPER "Build with libradosstriper support." OFF)
```

Comment:
Wouldn't this change the current upstream behavior?

Reply:
Yes-- looks like I thought the opposite of what we actually want-- fixed.